src/custom/state/orders/updaters/UnfillableOrdersUpdater.ts

```diff
@@ -24,7 +24,12 @@ async function _getOrderPrice(chainId: ChainId, order: Order, strategy: GpPriceS
   let amount, baseToken, quoteToken
 
   if (order.kind === 'sell') {
-    amount = order.sellAmount.toString()
+    // this order sell amount is sellAmountAfterFees..
```

Comment:
nit: can you use the imperative for comments? 

i.e. Use sell amount after fees for market price calculation. It includes the fee because...